hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/metrics/OzoneManagerDoubleBufferMetrics.java

```diff
@@ -44,6 +47,15 @@
       "number of transactions flushed in a single flush iteration till now.")
   private MutableCounterLong maxNumberOfTransactionsFlushedInOneIteration;
 
+  @Metric(about = "DoubleBuffer flushTime. This metrics particularly captures" +
+      " rocksdb batch commit time.")
+  private MutableRate flushTime;
+
+  @Metric(about = "Average number of transactions flushed in a single " +
+      "iteration")
+  private MutableGaugeLong avgFlushTransactionsInOneIteration;
```

Comment:
Can this be a MutableGaugeFloat instead of a MutableGaugeLong? We are losing some precision here because of the division and cast to Long.

Reply:
Thank You @avijayanhwx for the review. Address comments in the latest commit.